src/spath.c

```diff
@@ -14,18 +15,18 @@
 
 /* allocate size bytes, returns NULL if size == 0,
  * calls er_abort if allocation fails */
-void* spath_malloc(size_t size, const char* file, int line)
-{
-  void* ptr = NULL;
-  if (size > 0) {
-    ptr = malloc(size);
-    if (ptr == NULL) {
-      fprintf(stderr, "Failed to allocate %llu bytes @ %s:%d", (unsigned long long) size, file, line);
-    }
-  }
-  return ptr;
-}
-#define SPATH_MALLOC(X) spath_malloc(X, __FILE__, __LINE__);
+//void* spath_malloc(size_t size, const char* file, int line)
```

Comment:
And go ahead and delete these lines, too for cleanup

Reply:
Looks like there were a few more lines that were commented out. Those need to be cleaned up as well?